src/Compilers/Core/Portable/InternalUtilities/ISetExtensions.cs

```diff
@@ -17,6 +18,17 @@ public static bool AddAll<T>(this ISet<T> set, IEnumerable<T> values)
             return result;
         }
 
+        public static bool AddAll<T>(this ISet<T> set, ImmutableArray<T> values)
+        {
+            var result = false;
+            foreach (var v in values)
+            {
+                result |= set.Add(v);
```

Comment:
Even if you don't use the result, I think the proper result behavior is to return `true` if and only if all items were added successfully and false otherwise. That way others can use this helper and rely upon the result.